src/ooni/ooni_test_impl.hpp

```diff
@@ -110,12 +110,15 @@ class OoniTestImpl : public mk::NetTest, public mk::NonCopyable,
     }
 
     void open_report() {
+        if (output_filepath == "") {
+            output_filepath = generate_output_file_path();
+        }
```

Comment:
Removed the redundant `get_output_file_path()` accessor and moved its implementation here